app/scripts/models/auth_brokers/base.js

```diff
@@ -286,6 +286,10 @@ define([
        */
       emailVerificationMarketingSnippet: true,
       /**
+       * Should the view handle signed-in notifications from other tabs?
+       */
+      signedInNotificationHandler: true,
```

Comment:
Much better name, though I wonder if it can be made even more clear, something like `acceptSignInFromOtherTabs`?